### PR TITLE
Add `--inset` flag

### PR DESCRIPTION
**Pull request by krnik**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -77,6 +78,7 @@ const cli = meow(`
 	  --authentication="username:password"
 	  --launch-options='{"headless": false}'
 	  --dark-mode
+	  --inset=10,15,-10,15
```
Comment (sindresorhus) [inline cli.js:81]: Add an example with a single element too.

File: cli.js [hunk 2]
```diff
@@ -206,6 +211,25 @@ if (options.authentication) {
...
+
+	if (containsNaN || ![1, 4].includes(values.length)) {
+		console.error('The `inset` flag value is invalid, please refer to the flag examples');
+		process.exit(1);
+	}
+
+	if (values.length === 1) {
+		options.inset = values[0];
+	} else {
+		options.inset = ['top', 'right', 'bottom', 'left'].reduce((acc, key, index) => {
```
Comment (sindresorhus) [inline cli.js:226]: I prefer to not use `reduce`. Code is usually more readable with a `for-of` loop.

File: cli.js [hunk 3]
```diff
@@ -206,6 +211,25 @@ if (options.authentication) {
 	options.authentication = {username, password};
 }
 
+if (options.inset) {
+	const values = options.inset.split(',').map(chunk => Number.parseInt(chunk, 10));
+	const containsNaN = values.findIndex(number => Number.isNaN(number)) > -1;
```
Comment (sindresorhus) [inline cli.js:216]: `Array#every` would be a better choice here.

File: cli.js [hunk 4]
```diff
@@ -206,6 +211,25 @@ if (options.authentication) {
...
+
+	if (containsNaN || ![1, 4].includes(values.length)) {
+		console.error('The `inset` flag value is invalid, please refer to the flag examples');
```
Comment (sindresorhus) [inline cli.js:219]:
```suggestion
		console.error('Invalid `--inset` value');
```

File: cli.js [hunk 5]
```diff
@@ -43,6 +43,7 @@ const cli = meow(`
 	  --dark-mode              Emulate preference of dark color scheme
 	  --launch-options         Puppeteer launch options as JSON
 	  --overwrite              Overwrite the destination file if it exists
+	  --inset                  Set the inset of the screenshot relative to the viewport or \`--element\`. Accepts \`number\` or four comma-separated numbers for top, right, left and bottom values
```
Comment (sindresorhus) [inline cli.js:46]:
```suggestion
	  --inset                  Inset the screenshot relative to the viewport or \`--element\`. Accepts a number or four comma-separated numbers for top, right, left, and bottom.
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
The readme and the actual CLI output are not in sync.

**Comment by krnik** [comment]
Right, I missed this change. Should be synced now.

**Pull request merged**